### ISSUE TITLE
Warning: The 'no-circular-imports' rule requires type checking

### ISSUE THREAD

**Issue by ccorcos** [bcherny/tslint-no-circular-imports#30]
Not sure what to do about this warning... the plugin doesnt appear to be working.

**Comment by bcherny** [comment]
Google is your friend :)

https://palantir.github.io/tslint/usage/type-checking/

**Comment by ccorcos** [comment]
This is the command you have to run. Might make sense to put this in the README.

```sh
./node_modules/.bin/tslint --project tsconfig.json --config tslint.json --type-check
```

I also have a really large project so I had to run with this command:

```sh
node --max-old-space-size=8192 ./node_modules/.bin/tslint --project tsconfig.json --config tslint.json --type-check
```
